File: practica1/P1S3/Cliente.cpp
#include "FilterManager.cpp"

class Cliente
{
    private:
        FilterManager gestorFiltros;

    public:
        Cliente(FilterManager gestor)
        {
            gestorFiltros = gestor;
        }

        void Request()
        {
            gestorFiltros.ejecutarFiltros();
        }

};

File: practica1/P1S3/Java/src/practica1dss3/Cliente.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package practica1dss3;

/**
 *
 * @author USUARIO
 */
public class Cliente {
    private FilterManager gestorFiltros;
    
    public Cliente(FilterManager f){
        gestorFiltros = f;
    }
    
    public void Request(){
        gestorFiltros.ejecutarFiltros();
    }
}
